public/setting.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <title>タイピングゲーム</title>
+    <link rel="stylesheet" href="./stylesheets/setting.css" />
+
+    <script type="module" src="./javascripts/main.js"></script>
+  </head>
+
+  <body>
+    <div data-screen="setting">
+      <h2 class="settingItem">設定</h2>
```

Comment:
他がケバブケースだからここだけキャメルケースなのは不自然なので統一した方が良いと思います。

```suggestion
      <h2 class="setting-item">設定</h2>
```